src/main/java/org/alfresco/rest/api/DeletedNodes.java

```diff
@@ -70,4 +71,29 @@
      * @param archivedId
      */
     void purgeArchivedNode(String archivedId);
+
+    /**
+     * Download file content (or rendition content) via archived node.
+     *
+     * @param archivedId
+     * @param renditionId
+     *            - optional
+     * @param parameters
+     *            {@link Parameters}
+     * @return
+     */
+    BinaryResource readProperty(String archivedId, String renditionId, Parameters parameters);
```

Comment:
This method has a very generic name that mirrors the REST framework method, but here it should be specific -- `readContent(...)` or similar?